frank/radial_fitters.py

```diff
@@ -716,6 +722,8 @@ def fit(self, u, v, V, weights=1):
         pi_old = 0
         while (np.any(np.abs(pi - pi_old) > self._tol * pi) and
                count <= self._max_iter):
+            print('\r    Fit iteration {}'.format(count), end='', flush=True)
+
```

Comment:
Remove from this PR 

```suggestion
```

Reply:
I've already thought about and tested this, so there's no need to just remove it; I'm commenting for now.